client/allocrunner/taskrunner/template/template.go

```diff
@@ -574,6 +574,13 @@ func parseTemplateConfigs(config *TaskTemplateManagerConfig) (map[ctconf.Templat
 		ct.LeftDelim = &tmpl.LeftDelim
 		ct.RightDelim = &tmpl.RightDelim
 
+		// By default we pass a blacklist of functions to prevent
+		// task operators from bypassing client-task isolation.
+		// This protection can be disabled by the client config.
+		if !config.ClientConfig.EnableInsecureTemplateFunctions {
```

Comment:
would like to be able to disable `plugin` but keep `file` around - maybe expose a []string for the functions I want to blacklist manually as well?